pkg/find/find.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package find
+
+import (
+	"os"
+	"path/filepath"
+)
+
+type Names struct {
```

Comment:
Nit: `type Names` should be `Name` and `Finder.Name` should be `Finder.Names` as the struct contains a single name, while Finder.Name contains the channel where multiple names are passed.